gateway/operations/postobject.go

```diff
@@ -54,7 +55,7 @@ func (controller *PostObject) HandleCreateMultipartUpload(o *PathOperation) {
 	}
 	o.EncodeResponse(&serde.InitiateMultipartUploadResult{
 		Bucket:   o.Repository.Name,
-		Key:      o.Path,
+		Key:      path.WithRef(o.Path, o.Reference),
```

Comment:
I don't understand.  How did multipart uploads work without this?

Reply:
They work just fine if you don't pass the key that we're returning. Once you pass it back they'll fail.